frontend/src/Student/components/OtherMediaSubmissionForm.js

```diff
@@ -162,12 +166,17 @@ class OtherSubmissionForm extends Component {
   }
 
   renderShow = () => {
-    const { create, done, user, handleError } = this.props
+    const { create, done, user, handleError, handleHometown, handleDisplayName } = this.props
     const forShow = {
       id: this.props.data.show.id,
       name: this.props.data.show.name
     }
 
+    const defaultHometown = user.hometown || '';
+    const hometownNeeded = !user.hometown;
+    const defaultDisplayName = user.displayName || '';
+    const displayNameNeeded = !user.displayName;
```

Comment:
Wouldn't this be based off the checkbox?

Reply:
This should be the value of from formik

Reply:
addressed in newest commit